torch/onnx/_internal/exporter/_core.py

```diff
@@ -768,7 +766,7 @@ def _exported_program_to_onnx_program(
             },
         ),
         ir_version=9,
```

Comment:
Should we hide this for version control?

Reply:
Can you elaborate?

Reply:
I am thinking ir version should go with onnxscript version, as it's coded in onnxscript. It seems less informative when we hardcode it here. Would we have to compare ir version in debugging?

Reply:
IR version is mainly used for enabling the latest dtypes. Hard coding to 9 seems ok for this pytorch version as it has all the dtypes we need.